src/test/java/org/llorllale/cactoos/matchers/IsEntryTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) for portions of project cactoos-matchers are held by
+ * Yegor Bugayenko, 2017-2018, as part of project cactoos.
+ * All other copyright for project cactoos-matchers are held by
+ * George Aristy, 2018.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package org.llorllale.cactoos.matchers;
+
+import org.cactoos.map.MapEntry;
+import org.hamcrest.core.IsEqual;
+import org.hamcrest.core.IsNot;
+import org.junit.Test;
+
+/**
+ * Test for {@link IsEntry}.
+ *
+ * @since 1.0.0
+ * @checkstyle ClassDataAbstractionCoupling (100 lines)
+ */
+public final class IsEntryTest {
+    /**
+     * Negative case.
+     */
+    @Test
+    public void doesNotMatchEntry() {
+        new Assertion<>(
+            "must not match the entry",
+            new IsEntry<>(
+                new IsEqual<>("q"),
+                new IsEqual<>("w")
+            ),
+            new IsNot<>(new Matches<>(new MapEntry<>("k", "v")))
```

Comment:
@andreoss I missed this one, can you replace ALL the isnot(matches) with mismatches?

Reply:
@victornoel Replaced all which is mine. There is a second PR which deals with the rest.
I think that `Mismatch` should be relaxed and not require `expected` and `actual` messages.

Reply:
@andreoss this is another problem, open an issue if you want, but the problem with this proposal is that it opens the door to incomplete tests which is a problem IMHO.

Reply:
@andreoss (when I said ALL, I really meant all in this PR of course)